e2e-etl-python/files/Makefile

```diff
@@ -48,27 +48,7 @@ prep-test:
 .PHONY: test
 # Run (pre-deployment) tests.
 test: install-test-deps
-	@$(call check_aws_credentials)
-
-	# output aws account and user id for testing
-	aws sts  get-caller-identity --output text | tee $(INSTALL_REPORT_HOME)/aws_testing_account.log
-
-	# Remove any previously created Terraform test artefacts.
-	for dir in .terraform terraform.tfstate.d; do \
-		find test/fixtures -name $$dir -print0 | xargs -0 rm -rf; \
-	done \
-
-	inspec_profiles=$$(ls -1 ./test/integration); \
-	for fdir in $$inspec_profiles; do \
-		mkdir -p test/integration/$$fdir/files ; \
-		./.venv/bin/python3 ./.venv/bin/hcl2tojson test/fixtures/$$fdir/main.tf  test/integration/$$fdir/files/main.json; \
-	done \
-
-	# See https://github.com/test-kitchen/test-kitchen/issues/1436 for why a simple `bundle exec kitchen test` is not an option.
-	for suite in $$(bundle exec kitchen list --bare); do \
-		bundle exec kitchen verify $$suite || { bundle exec kitchen destroy $$suite; exit 1; }; \
-		bundle exec kitchen destroy $$suite; \
-	done
+	#Removed for the e2e-etl-python QS
```

Comment:
this comment won't make sense to the end user and doesn't provide any context or important information to them